### DISCUSSION TITLE
New selection features, support for more navigation keys

### DISCUSSION THREAD

**Discussion by FlorianCassayre** [iddan/react-spreadsheet#306] [Ideas]
This library looks very promising, thank you!

Here are a couple of desirable enhancement ideas that I can think of:
* Ability to move selected data
* Autofill (the cross that appears at the bottom right of a selection, and allows you to extrapolate the selected data)
* Support navigation keys (home, end, page up/down)

Feel free to split it into multiple tickets if needed, or argue that these features can be implemented at the user level.

**Comment by iddan** [comment]
1. Great idea
2. Autofill is tracked over here #302 
3. Great idea as well
